src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,19 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        final DriverService driverService
+                = (DriverService) injector.getInstance(DriverService.class);
+        final ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        final CarService carService
+                = (CarService) injector.getInstance(CarService.class);
```

Comment:
let's move the services declaration outside of the method, and test the methods that we have implemented here, as it says in the task description